keel-sql/src/main/resources/db/changelog/20191202-create-diff-fingerprint-table.yml

```diff
@@ -0,0 +1,48 @@
+databaseChangeLog:
+  - changeSet:
+      id: create-diff-fingerprint-table
+      author: emjburns
+      changes:
+        - createTable:
+            tableName: diff_fingerprint
+            columns:
+              - column:
+                  name: resource_id
+                  type: varchar(255)
+                  constraints:
+                    nullable: false
+              - column:
+                  name: first_detection_time
+                  type: bigint
```

Comment:
Is that the right data type for an `Instant`?

Reply:
It's a long (`clock.instant().toEpochMilli()`)

Reply:
Why not just use a timestamp, then you don't have to use `toEpochMilli`? Same as e.g. `resource_event` table.

Reply:
Because @asher said that all the other services store time as the long value. Personally I find using `toEpochMilli()` is so much easier than reasoning about time and timestamps and timestamp format.

Reply:
I would ❤️ if we converted all timestamp persistence to epoch milli's so we don't have to consider TZ's except where we want to and maintain millisecond precision (I seem to remember that mysql timestamp's are second precision only but mot 💯 on that.)

Reply:
Hmm, yeah, I was forgetting the hoops we have to jump thru to convert a tz-agnostic `Instant` into a `LocalDateTime` to get it into the table. Pretty sure we wouldn't have that issue with Postgres… grumble, grumble.

Reply:
I guess my main / only issue with using epoch millis is that it's harder to interpret raw SQL query results.